### PR TITLE
Fixes #730 - handles when translations are empty

### LINKED ISSUE #730: Keyword translation needs to handle over quota state
When the Google Translate API quota is reached (such as when billing is disabled) a call to the API returns an empty array at `result.data.translations`. https://github.com/codeforamerica/ohana-web-search/blob/master/app/services/keyword_translator.rb#L49 has `translations.first.translatedText`, which fails in an over-quota scenario because `translations.first` is nil. 
https://github.com/codeforamerica/ohana-web-search/blob/master/app/services/keyword_translator.rb#L48 should be rewritten to something like:

```
return keyword if translations.nil? || translations.length == 0
```

@monfresh you might have a better solution for this?

### PR DESCRIPTION
**Pull request by anselmbradford**
Fixes #730 - handles when translations return an empty result set, such
as when the account is over quota (billing is not enabled).

### CODE REVIEW

**Comment by anselmbradford** [comment]
@monfresh Ready for review and merge. Thx!

File: app/services/keyword_translator.rb [hunk 1]
```diff
@@ -45,7 +45,7 @@ def translations
   end
 
   def translated_keyword
-    return keyword if translations.nil?
+    return keyword if translations.nil? || translations.length == 0
```
Comment (monfresh) [inline app/services/keyword_translator.rb]: We should use the Rails `blank?` method which automatically checks for either nil or empty array in this case.
So, `return keyword if translations.blank?` should do it.

Comment (monfresh) [inline app/services/keyword_translator.rb]: Also, just for informational purposes, you can use `empty?` in Ruby to check if an array, string, or hash is empty.

**Comment by anselmbradford** [comment]
Cool, thanks. I'll update that and merge it.

**Pull request merged**